<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Room&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="roomComboBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="scheduleTableWidget">
      <row>
       <property name="text">
        <string>08:00 - 08:45</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>08:55 - 09:40</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>09:50 - 10:35</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10:45 - 11:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11:40 - 12:25</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12:35 - 13:20</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13:30 - 14:15</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14:25 - 15:10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15:20 - 16:05</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16:15 - 17:00</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Mon</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tue</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wed</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Thu</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fri</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRooms"/>
    <addaction name="actionTeachers"/>
    <addaction name="actionClasses"/>
    <addaction name="actionGroups"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRooms">
   <property name="text">
    <string>Rooms...</string>
   </property>
  </action>
  <action name="actionTeachers">
   <property name="text">
    <string>Teachers...</string>
   </property>
  </action>
  <action name="actionGroups">
   <property name="text">
    <string>Groups...</string>
   </property>
  </action>
  <action name="actionClasses">
   <property name="text">
    <string>Classes...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
